client/src/periodUtils.js

```diff
@@ -11,7 +11,8 @@ export const RECURRENCE_TYPE = {
   SEMIMONTHLY: "semimonthly",
   MONTHLY: "monthly",
   QUARTERLY: "quarterly",
-  SEMIANNUALY: "semiannualy"
+  SEMIANNUALLY: "semiannually",
```

Comment:
if you do so, you will need a migration to refactor any existing json in the database fields with assessment definitions.
That, or you can ask @dspCarlson how many such instances of semianually are in the db - if any.

Reply:
Hmmm, right. I will revert this.